robot_lib.py

```diff
@@ -151,6 +152,37 @@ def dryGripper():
     daq_lib.gui_message("Dry gripper failed! " + e_s)
     setPvDesc("warmupThreshold",saveThreshold)          
     
+def DewarRefill(hours):
+  global _dewarRefillThread
+  seconds = int((hours * 60 * 60))
+  _dewarRefillThread = Thread(target=_dewarRefillTask, args=(seconds,0))
+  _dewarRefillThread.start()
+
+def _dewarRefillTask(seconds, flag):
```

Comment:
`flag` is not used so should be removed

Reply:
Fixed with 61ad2c5ffb84804c3b0f23ff3245f9952b4c5fc0